Britannia/localization/english/BTWK_storyline_events_l_english.yml

```diff
@@ -1,31 +1,58 @@
- BTWK_story_event.0005.a:0 "Hail to King Mordred and Lord Arthur!"
- BTWK_story_event.0005.b:0 "Let us hope Arthur drives the Saxons back to whence they came!"
- BTWK_story_event.0005.c:0 "They are weak. We will conquer their lands, no matter their leader."
- BTWK_story_event.0005.d:0 "This is interesting news."
-
- BTWK_story_event.0006.t:0 "From One Pendragon to Another"
- BTWK_story_event.0006.desc:0 "Your grandfather, King Uther Pendragon, has died. Prince Arthur, his bastard son and your uncle, has returned to Dumnonia from the Kingdom of Benoic upon hearing of Uther's death, where he had been sent to assist King Ban in his wars against the Franks. Upon his return, your current regent appointed him the Prince of Durnac. He has been hailed, like his father, as #italic Pendragon #. "
- BTWK_story_event.0006.a:0 "A loyal vassal... or is he plotting for my throne?"
- BTWK_story_event.0006.b:0 "PLACEHOLDER TEXT - NEED SOMETHING APPROPRIATE TO PROMPT TAG SWITCH" 
-
- BTWK_story_event.0007.t:0 "Arthur Pendragon"
- BTWK_story_event.0007.desc:0 "Arthur Pendragon is a man of many virtues and many flaws. Banished from Dumnonia after being blamed by Uther for the death of his only legitimate son, Mordred, in battle, Arthur served in the court of his brother-in-law, King Budic of Broceliande. Later, he found himself as aide to the neighbouring King of Benoic, Ban, defending against the hordes of barbarous Franks. With his father's passing, however, Arthur was obliged to return from Armorica to ensure the stablilty of the realm for his infant nephew, Mordred ap Mordred. Will you fulfil your oath to Uther as Mordred's protector, or instead take #italic other steps# to secure Dumnonia against the inevitable Saxon onslaught?"
- BTWK_story_event.0007.a:0 "The fate of Dumnonia lies with you."
-
- BTWK_story_event.0008.t:0 "W"
- BTWK_story_event.0008.desc:0 "X"
- BTWK_story_event.0008.desc_arthur_regent_1:0 "Y"
- BTWK_story_event.0008.desc_arthur_regent_2:0 "Z"
- BTWK_story_event.0008.a:0 "A"
- BTWK_story_event.0008.b:0 "B"
+
+ BTWK_story_event.0101.t:0 "The Winter King"
+ BTWK_story_event.0101.desc:0 "Your one and only heir is your newborn grandson, Mordred, after your only legitimate son - his namesake - was killed in battle some months prior. The babe's crippled leg - a bad omen - only helps to sow the seeds of discontent. Without sufficient preparation for your death, and his ascension to rule, this surely will spell disaster for Dumnonia. \n\nAs High King of Britannia, you have the power to convene 'High Councils' in times of peril. The last of these was over 80 years ago, but with the threat ever-increasing encroachment by the Saxons, and British warlords forever infighting, you see no other option to bring peace to Britannia once more. Messengers have been sent you would like to meet them in Glevum."
```

Comment:
"and his ascension to rule" - something sounds a bit off about this phrasing here
"authority to convene" instead of "power to convene"?
"80 years ago" - was this something stated in the books? Doesn't seem quite right to me, it would be around 400 AD.
"you see no other option to bring peace to Britannia once more" - seems a bit lofty considering what they're actually deciding at the council TBH, I would just go for "ensure stability" or something.

Reply:
1. "Without sufficient preparation for your death - #italic and#! Mordred's ascension - disaster will surely befall Dumnonia."
2. Amended
3. Amended in a previous commit (somehow?)
4. "...and British warlords forever infighting, this could be a chance to bring renewed stability."?

Reply:
1. Maybe "Mordred's premature ascension" ?
4. Sure